src/user-guide/running-workflows/scheduler-log-files.rst

```diff
@@ -1,7 +1,112 @@
 .. _Scheduler Logs:
+.. _user-guide.log_files:
 
-Scheduler Logs
---------------
+Workflow Logs
+=============
+
+Cylc produces a number of log files which record information about the
+wortkflow, how it was installed, what tasks have run, their outcomes and more.
```

Comment:
I think it's worth ordering this sentence by the order in which Cylc logs stuff.
```suggestion
Cylc produces log files which record information how the workflow was
installed, configured when started or changed, and about tasks run.
```

Reply:
Missing an "about" between "information" and "how"

Reply:
Sorry, "and about tasks run" doesn't make sense.

"and the tasks that have run and their outcomes"?

Reply:
Have edited my proposed change. I agree.